docusaurus/docs/guides/Identity_Providers/_category_.yml

```diff
@@ -0,0 +1,5 @@
+label: BrowZer Identity Providers
```

Comment:
I like the shorter "BrowZer" for the sidebar category label, especially on mobile.

Reply:
how 'bout we go with `BrowZer IdPs`?  It will be shorter while still indicating the main topic, which is all about IdPs for BrowZer.

Reply:
it can't be "just" browzer for sure. BrowZer IdPs is probably fine. i don't agree that on mobile it makes a difference. for me it would be fine spelled out